optaplanner-examples/src/main/java/org/optaplanner/examples/coachshuttlegathering/domain/location/RoadLocationArc.java

```diff
@@ -67,4 +67,12 @@ public void setShuttleDuration(int shuttleDuration) {
     // Complex methods
     // ************************************************************************
 
+    @Override
+    public String toString() {
+        return "Road arc [coach: "
```

Comment:
Can we change this to be in the same style as the rest of the domain classes toString() methods?